src/constraints/capacity.jl

```diff
@@ -360,7 +395,53 @@ function _append_capacity_data_to_indices(connection, table_name)
         LEFT OUTER JOIN assets_profiles AS avail_profile
             ON cons.asset = avail_profile.asset
             AND expr_avail.commission_year = avail_profile.commission_year
+            AND avail_profile.profile_type = 'availability'
+        WHERE asset.investment_method = 'compact'
+        GROUP BY cons.id
+        ORDER BY cons.id
+        ",
+    )
+end
+
+# - The simple method selects the simple or the none investment method
+# - and do not aggregate the available capacity indices, because there will be only 1.
+# - It is a choice the the none method takes the simple formulation (can also take the compact formulation)
+function _append_capacity_data_to_indices_simple_method(connection, table_name)
+    return DuckDB.query(
+        connection,
+        "SELECT
+            ANY_VALUE(cons.id) AS id,
+            ANY_VALUE(cons.asset) AS asset,
+            ANY_VALUE(cons.year) AS year,
+            ANY_VALUE(cons.rep_period) AS rep_period,
+            ANY_VALUE(cons.time_block_start) AS time_block_start,
+            ANY_VALUE(cons.time_block_end) AS time_block_end,
+            ANY_VALUE(expr_avail.id) AS avail_indices,
+            ANY_VALUE(expr_avail.commission_year) AS avail_commission_year,
+            ANY_VALUE(expr_avail.initial_units) AS avail_initial_units,
+            ANY_VALUE(avail_profile.profile_name) AS avail_profile_name,
+            ANY_VALUE(asset.capacity) AS capacity,
+            ANY_VALUE(asset.investment_method) AS investment_method,
+            ANY_VALUE(asset_commission.investment_limit) AS investment_limit,
+            ANY_VALUE(assets_profiles.profile_name) AS profile_name,
+        FROM cons_$table_name AS cons
+        LEFT JOIN asset
+            ON cons.asset = asset.asset
+        LEFT JOIN asset_commission
+            ON cons.asset = asset_commission.asset
+            AND cons.year = asset_commission.commission_year
+        LEFT JOIN expr_available_asset_units_simple_method AS expr_avail
+            ON cons.asset = expr_avail.asset
+            AND cons.year = expr_avail.milestone_year
+        LEFT OUTER JOIN assets_profiles
+            ON cons.asset = assets_profiles.asset
+            AND cons.year = assets_profiles.commission_year
             AND assets_profiles.profile_type = 'availability'
+        LEFT OUTER JOIN assets_profiles AS avail_profile
+            ON cons.asset = avail_profile.asset
+            AND expr_avail.commission_year = avail_profile.commission_year
+            AND avail_profile.profile_type = 'availability'
+        WHERE asset.investment_method in ('simple', 'none')
         GROUP BY cons.id
         ORDER BY cons.id
         ",
```

Comment:
Since there is only 1 here, you shouldn't need to `GROUP BY` - which also means you don't need `ANY_VALUE`.